tests/mgbench/datasets.py

```diff
@@ -45,7 +45,9 @@ def __init__(self, variant=None):
             variant = self.DEFAULT_VARIANT
         if variant not in self.VARIANTS:
             raise ValueError("Invalid test variant!")
-        if (self.FILES and variant not in self.FILES) and (self.URLS and variant not in self.URLS):
+        if (self.FILES and variant not in self.FILES) and (
+            self.URLS and variant not in self.URLS
+        ):
```

Comment:
I guess the formatting is automatic. I think we all need to use the same configuration for that, my automatic formatting seems to give another layout. I'll look at it

Reply:
I synced with @jbajic about this. Thanks for the tip.